doc/sql.extensions/README.named_arguments.md

```diff
@@ -0,0 +1,52 @@
+# Named arguments for function and procedure calling (FB 6.0)
+
+Named arguments allows you to specify function and procedure arguments by their names, rather than only by their positions.
+
+It is especially useful when the routine have a lot of parameters and you want to specify them in arbitrary order or not specify some of them who have default values.
+
+As the positional syntax, all arguments without default values are required to be present in the call.
+
+It's currently not possible to mix positional and named arguments in the same call.
+
+## Syntax
+
+```
+<function call> ::=
+    [<package name> .] <function_name>( [<arguments>] )
+
+<procedure selection> ::=
+    [<package name> .] <procedure_name> [( <arguments> )]
+
+<execute procedure> ::=
+    EXECUTE PROCEDURE [<package name> .] <procedure name>
+    [{ (<arguments>) | <arguments> }]
+    [RETURNING_VALUES ...]
+
+<arguments> ::=
+    <named arguments>
+    <positional arguments>
+
+<named arguments> ::=
+    <named argument> [ {, <named argument>}... ]
+
+<named argument> ::=
+    <argument name> => <value>
```

Comment:
For me it looks a bit confusing, like assigning name to the value (name => value).
Does you consider to reuse existing `TOK_BIND_PARAM` ( `:=` ) instead ?
I understand that `=>` used in Oracle, but it not means we should copy it :)

Reply:
I see no reason not to use both.

Reply:
In addition to be used by Oracle and PgSQL, `=>` is SQL standard, so it's not confusing.
I also remember to suggest it when you created `:=`, but in fact as things are not exactly identical, I see no problem to use each of them in their respective places.